Test/Graph/test_graph.py

```diff
@@ -368,33 +378,33 @@ def test_adjacency_list():
             assert set(adl[i]) in neigh
 
 
-def test_grid_color_pbc():
-    # compute length from iterator
-    count = lambda it: sum(1 for _ in it)
-
-    g = Grid([4, 4], pbc=True, color_edges=True)
-    assert count(g.edges(color=0)) == 16
-    assert count(g.edges(color=1)) == 16
-    assert g.n_edges == 32
-
-    g = Grid([4, 2], pbc=True, color_edges=True)
-    assert count(g.edges(color=0)) == 8
-    assert count(g.edges(color=1)) == 4
-
-    g = Grid([4, 2], pbc=False, color_edges=True)
-    assert count(g.edges(color=0)) == 6
-    assert count(g.edges(color=1)) == 4
-
-    with pytest.raises(ValueError, match="Directions with length <= 2 cannot have PBC"):
-        g = Grid([2, 4], pbc=[True, True])
-
-    g1 = Grid([7, 5], pbc=False)
-    g2 = Grid([7, 5], pbc=[False, False])
-    assert sorted(g1.edges()) == sorted(g2.edges())
-
-    g1 = Grid([7, 5], pbc=True)
-    g2 = Grid([7, 5], pbc=[True, True])
-    assert sorted(g1.edges()) == sorted(g2.edges())
+# def test_grid_color_pbc():
```

Comment:
Maybe better to remove this instead of commenting it out. (If we add this back later, the test is still there in the git history if needed.)

Reply:
Though the code for coloured graphs is still there, and with this removal we are no longer testing it (according to codecov)

Can't we adapt the tests to test directly `graph.NetworkX`?

Reply:
That's strange — I've just resurrected an test for coloured edge graphs a bit further down. What isn't tested?

BTW, this is why I didn't remove it, it will be more directly useful if it's actually there, and it's also a reminder to sort out coloured edges...

Reply:
I was going through the codecov report. If you look at [this line](https://app.codecov.io/gh/netket/netket/compare/724/changes#D1L73) it was previously covered and now it's not anymore.